src/g_graph.c

```diff
@@ -479,6 +479,10 @@ static void graph_goprect(t_glist *x, t_symbol *s, int argc, t_atom *argv)
     else
         /* glist_redraw() won't remove "ghost objects" */
         canvas_redraw(glist_getcanvas(x));
+
+        /* always update cords on parent if visible. */
+    if (x->gl_owner && !x->gl_isclone && glist_isvisible(x->gl_owner))
+        canvas_fixlinesfor(x->gl_owner, &x->gl_obj);
```

Comment:
As you said, this is only an issue if the abstraction window is open, so I think the correct solution would be:
```
if (x->gl_havewindow)
    {
        glist_redraw(x);
        /* possibly fix patch cords on parent canvas */
        if (x->gl_owner && !x->gl_isclone && glist_isvisible(x->gl_owner))
            canvas_fixlinesfor(x->gl_owner, &x->gl_obj);
    }
    else
        /* glist_redraw() won't remove "ghost objects" */
        canvas_redraw(glist_getcanvas(x));
```
This seems to work fine for me.

(Note that `canvas_redraw(glist_getcanvas(x));` already redraws the whole parent canvas, including the patch cords.)